Source/reactions/Castro_react.cpp

```diff
@@ -461,6 +476,18 @@ Castro::react_state(Real time, Real dt)
             }
 #endif
 
+            // Convert the current state to primitive data.
+            // This state is U* = U_old + dt A where A = -div U + S_hydro.
+
+            Array1D<Real, 0, NQ-1> q_noreact;
+            Array1D<Real, 0, NQAUX-1> qaux_noreact;
+
+            hydro::conservative_to_primitive(i, j, k, U_new,
```

Comment:
should this be SDC_react instead of U_new?

Reply:
No -- in this version of the code, we never need to do the copy of the state into SDC_react. At this point in the code flow, U_new still represents U_old + dU_hydro + dU_sources. It's only after calling `burner` that this is modified. So this allows us to do what we were doing before without ever needing to make a temporary MultiFab copy of U_new.